crates/namada/src/ledger/native_vp/masp.rs

```diff
@@ -390,7 +390,7 @@ where
         }
 
         // The Sapling value balance adds to the transparent tx pool
-        let mut transparent_tx_pool = shielded_tx.sapling_value_balance();
+        let transparent_tx_pool = shielded_tx.sapling_value_balance();
```

Comment:
Since `transparent_tx_pool` is only used inside `fn validate_transparent_bundle`, maybe this variable declaration could be moved into `fn validate_transparent_bundle`? If doable, this would reduce the number of parameters taken by `fn validate_transparent_bundle`.

Reply:
You're right! Done in a3bb632a1f40700abae9027a80cc9fcd88725f70